trino-connector/src/main/java/com/datastrato/gravitino/trino/connector/catalog/CatalogConnectorFactory.java

```diff
@@ -52,23 +46,6 @@ public CatalogConnectorContext loadCatalogConnector(
     }
 
     // Avoid using the same builder object to prevent catalog creation errors.
-    builder = builder.clone();
-
-    try {
-      Connector internalConnector =
```

Comment:
Why do we remove this logic here?

Reply:
The `getFullCatalogName` function is located in CatalogConnectorManager.java. We should use it there since it relies on CatalogConnectorManager.java. I moved the code to this file to minimize dependencies